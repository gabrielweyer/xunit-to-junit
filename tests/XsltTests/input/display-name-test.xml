<?xml version="1.0" encoding="utf-8"?>
<assemblies timestamp="12/31/2017 14:59:12">
  <assembly name="Contoso.Hello.HelloTests.dll" environment="64-bit .NET Standard [collection-per-class, parallel (4 threads)]" test-framework="xUnit.net 2.3.1.3858" run-date="2017-12-31" run-time="14:59:12" total="1" passed="1" failed="0" skipped="0" time="0.181" errors="0">
    <errors />
    <collection total="1" passed="1" failed="0" skipped="0" name="Test collection for Contoso.Hello.HelloTests.EvenMoreUselessTests" time="0.071">
      <test name="My Name" type="Contoso.Hello.HelloTests.EvenMoreUselessTests" method="WhenDoWork_ThenSomeSweetJson" time="0.0707464" result="Pass" />
    </collection>
  </assembly>
</assemblies>