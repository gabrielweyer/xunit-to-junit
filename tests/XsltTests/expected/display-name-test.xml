<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite name="Contoso.Hello.HelloTests.dll" tests="1" failures="0" skipped="0" errors="0" time="0.181" timestamp="2017-12-31T14:59:12">
    <testcase name="My Name" classname="Contoso.Hello.HelloTests.EvenMoreUselessTests" time="0.0707464" />
  </testsuite>
</testsuites>